host-bmc/host_pdr_handler.cpp

```diff
@@ -1112,7 +1126,7 @@ void HostPDRHandler::_setHostSensorState()
                         std::cerr
                             << " Error Invalid data, Invalid sensor offset,"
                             << " SensorId=" << sensorId << std::endl;
-                        return;
+                        continue;
```

Comment:
If this error occurs for one composite sensor it will occur for all so better to return than continue.

Reply:
fixed here - https://github.com/ibm-openbmc/pldm/pull/233